components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -184,5 +185,6 @@ export class TeamManagementComponent implements OnInit, OnDestroy {
     this.creatingTeam = false;
     this.createTeamForm.reset();
     this.createV1TeamForm.reset();
+    this.conflictErrorEvent.emit(false);
```

Comment:
this resets the conflict error on the create-object-modal, so we don't see `Object with ID "" already exists` next time we open the modal. I added this to `project-list` and `api-token-list` as well